src/main/java/com/feedzai/commons/sql/abstraction/dml/result/ResultIterator.java

```diff
@@ -167,8 +186,12 @@ public ResultIterator(PreparedStatement statement) throws DatabaseEngineExceptio
         }
     }
 
+    public ResultSet getResultSet() {
```

Comment:
javadoc missing.

Reply:
Removed this method, since it's not really necessary.